code/web/src/modules/user/Profile.js

```diff
@@ -42,32 +42,59 @@ const Profile = (props) => (
       </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
+        <p style={{ marginBottom: '0.5em' }}>{props.user.details.description ? props.user.details.description : "You don't have a description yet..."}</p>
         {/*Description text should be dynamic from user details ie props.user.details.description */}
       </GridCell>
```

Comment:
We need to make sure to do a sweep before 5 and remove any unnecessary comments.